app/views/pages/knock_out.html.erb

```diff
@@ -0,0 +1,29 @@
+<% title = t("views.pages.knock_out.title") %>
+<% content_for :page_title, title %>
+
+<section class="slab question-layout <%= controller_name.gsub("_", "-") %>-outer">
+  <div class="grid">
+    <h1 class="h2"><%= title %></h1>
+    <p><%= t("views.pages.knock_out.locked_due_to") %></p>
+    <% if @intake.present? && @intake&.permanently_locked_at.nil? %>
+      <p><%= t("views.pages.knock_out.one_hour")%></p>
+    <% end %>
+    <p><%= t("views.pages.knock_out.also_visit_html", year: session[:year_selected] || "")%></p>
+    <% if @intake.present? %>
+      <div class="bulleted">
+        <ul>
+          <li><a href="https://azdor.gov/forms/other-forms/request-copies-tax-documents"><%= t("views.pages.knock_out.az_dor") %></a></li>
+          <li><a href="https://www.tax.ny.gov/help/contact/get-copy-of-return.htm"><%= t("views.pages.knock_out.ny_dor") %></a></li>
```

Comment:
we dont want to show `ny` if they selected 2024, right?

Reply:
good catch